modules/serverless-installing-cli-macosx.adoc

```diff
@@ -0,0 +1,32 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative-client.adoc
+
+[id="installing-cli-macosx_{context}"]
+= Installing the `kn` CLI for Mac OS X
+
+`kn` for Mac OS X is provided as a `tar.gz` archive. You can download it from the link:TBD[Red Hat Customer Portal]. 
+
+.Procedure
+
+. Log in to the Red Hat Customer Portal. You must have an active subscription to access the downloads page.
+
+. Download the link:TBD[CLI] from the Red Hat Customer Portal.
```

Comment:
`Download the CLI

(remove the text 'from the Red Hat Customer Portal.' as we are not redirecting users there)